pkg/web/routes.go

```diff
@@ -146,6 +147,7 @@ func (s *Server) setupRoutes() (err error) {
 			transactions.GET("/:id", authorize(permiss.TravelRuleView), s.TransactionDetail)
 			transactions.PUT("/:id", authorize(permiss.TravelRuleManage), s.UpdateTransaction)
 			transactions.DELETE("/:id", authorize(permiss.TravelRuleDelete), s.DeleteTransaction)
+			transactions.GET("/:id/accept", authorize(permiss.TravelRuleView), s.TransactionAcceptDetail)
```

Comment:
Is this a preview route or does this actually trigger the accept? If the former, we should change the name to "Preview" if the latter, we should change it as follows:

```suggestion
			transactions.GET("/:id/accept", authorize(permiss.TravelRuleManage), s.AcceptTransaction)
```

Reply:
This will be the preview.